### ISSUE TITLE
Macro.to_string truncates AST with long binaries (> 4096 chars)

### ISSUE THREAD

**Issue by smetana** [elixir-lang/elixir#7545]
Macro.to_string uses Inspect and Inspect has the printable_limit option set to 4096 for binaries by default. So all long binaries in AST are truncated on Macro.to_string

```elixir
quote(do: unquote(String.duplicate("a", 4097)))
|> Macro.to_string()
```
The result is invalid elixir code

**Comment by net** [comment]
I believe this was fixed in f207745.

**Comment by josevalim** [comment]
Yup, fixed recently. I will backport it. :)
